supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/GenomicsFactory.scala

```diff
@@ -16,7 +17,7 @@ import mouse.all._
 
 import scala.collection.JavaConverters._
 
-case class GenomicsFactory(applicationName: String, authMode: GoogleAuthMode, endpointUrl: URL) extends PipelinesApiFactoryInterface
+case class GenomicsFactory(applicationName: String, authMode: GoogleAuthMode, endpointUrl: URL)(implicit localizationConfiguration: LocalizationConfiguration) extends PipelinesApiFactoryInterface
```

Comment:
Very-ToL: I've been burned so may times by implicits, including compile time, that I cringe at them. However this implementation is very "Scala" and matches multiple other libraries that I've seen, so LGTM.

Reply:
Yeah I thought this was going to be controversial, you're not the first one to bring this up 😄 . We do this in other places where people don't seem to have a problem with it (like all our database methods have an implicit execution context), and this seemed similar to me. I can see that it could potentially cause trouble if we had to choose between multiple configurations and the wrong one would implicitly be picked up (is that what's burned you before ?). Here there is only one of those per backend instance so we really should not be in that case.
But again people always have stronger opinions than me about this so happy to make it non-implicit.

Reply:
I share Khalid's disease with implicits in general but that doesn't rise to the level of a 'strong opinion'. 🙂